### PR TITLE
Update yarn.lock after kotlin upgrade

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Update yarn.lock after kotlin upgrade

### CODE REVIEW

**Review by nulls — APPROVED** [review]

**Review by DrAlexD — APPROVED** [review]

**Comment by kgevorkyan** [comment]
🥴

Capture

**Comment by kgevorkyan** [comment]
Local yarn require update, but CI's not 😐

**Review by Cheshiriks — APPROVED** [review]

**Review by akuleshov7 — APPROVED** [review]

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -207,3 +207,4 @@ aws-sdk-netty-nio = { module = "software.amazon.awssdk:netty-nio-client" }
 
 # vulnerability
 cosv4k = { module = "com.saveourtool.cosv4k:cosv4k", version.ref = "cosv4k" }
+
```
Comment (akuleshov7) [inline gradle/libs.versions.toml:210]: just an empty line?

Comment (kgevorkyan) [reply gradle/libs.versions.toml:210]: in local deployment we need to call `kotlinYarnUpdate` task to make `yarn.lock` be upgraded

however, CI fails and cant recognize the `yarn` updates

Thus, I change `lib.toml` in aim to trigger CI task for updating the `yarn.lock`, but diff is empty 😐

**Pull request closed without merging**